pkg/ui/page/projects.go

```diff
@@ -98,8 +126,10 @@ func (p *ProjectList) createComponents() {
 	// Create the layout.
 	p.layout = tview.NewFlex().
 		SetDirection(tview.FlexRow).
+		// AddItem(p.projectsTable, 0, 1, true).
 		AddItem(p.projectsTable, 0, 1, true).
-		AddItem(p.usage, 1, 1, false)
+		AddItem(p.usage, 1, 1, false).
+		AddItem(p.filterInputField, 1, 1, true)
```

Comment:
The focus by default I guess that shouldn't be true, thoughts?

Reply:
You're right! Will rebase and force-push!

Reply:
Actually, I'm not gonna rebase and force push, because then this converstation will make no sense to anyone from the outside.
I'll make this separate commits referencing your comment in this _discussion_.